cookbook/core/control_flow/subworkflows.py

```diff
@@ -81,17 +68,17 @@ def parent_wf(a: int) -> Tuple[int, str, str]:
 
 
 # %%
-# You can execute subworkflows locally
+# You can execute subworkflows locally.
 if __name__ == "__main__":
     print(f"Running parent_wf(a=3) {parent_wf(a=3)}")
 
 
 # %%
-# Example 2:
-# ^^^^^^^^^^
-# You can also nest subworkflows in other subworkflows as shown in the following example. Also note, how workflows
+# Nest Subworkflows in Other Workflows
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Can we have this as a sub-heading beneath "Why Should I use ..."? Will that look ok?